src/triage/component/results_schema/alembic/versions/5dd2ba8222b1_add_run_type.py

```diff
@@ -0,0 +1,54 @@
+"""add run_type
+
+Revision ID: 5dd2ba8222b1
+Revises: 079a74c15e8b
+Create Date: 2021-07-22 23:53:04.043651
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '5dd2ba8222b1'
+down_revision = '079a74c15e8b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('experiment_runs', sa.Column('run_type', sa.Text(), nullable=True), schema='triage_metadata')
+
+    op.add_column('experiment_runs', sa.Column('run_hash', sa.Text(), nullable=True), schema='triage_metadata')
+    op.drop_column('experiment_runs', 'experiment_hash', schema='triage_metadata')
+    op.execute("ALTER TABLE triage_metadata.experiment_runs RENAME TO triage_runs")
+
+    op.create_table('retrain',
+        sa.Column('retrain_hash', sa.Text(), nullable=False),
+        sa.Column('config', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
+        sa.Column('prediction_date', sa.DateTime(), nullable=True),
+        sa.PrimaryKeyConstraint('retrain_hash'),
+        schema='triage_metadata',
+    )
+
+    op.alter_column('models', 'built_in_experiment_run', nullable=False, new_column_name='built_in_triage_run', schema='triage_metadata')
```

Comment:
maybe add here (to avoid any potential for data loss):
```
op.execute("CREATE TABLE triage_metadata.deprecated_models_built_by_experiment AS SELECT model_id, model_hash, built_by_experiment FROM triage_metadata.models")
```